File: primer_punto.cpp
#include <iostream>
#include <iomanip>  

using namespace std;

int main() {
    double numero;
  
    cout << "Introduce un numero: ";
    cin >> numero;

    double suma = numero + 10;            
    double resta = numero - 5;            
    double multiplicacion = numero * 3;   
    double division = numero / 2;         
    double porcentaje = numero * 0.20;    
    
    cout << fixed << setprecision(2);  

    cout << "Suma (número + 10): " << suma << endl;
    cout << "Resta (número - 5): " << resta << endl;
    cout << "Multiplicación (número * 3): " << multiplicacion << endl;
    cout << "División (número / 2): " << division << endl;
    cout << "El 20% de ese número: " << porcentaje << "%" << endl;

    return 0;
}

File: primer_punto.py
#solicutar numeros
numero1 = float(input("Dame un numero"))
numero2 = float(input("Dame otro numero"))

#operaciones correspondientes
suma = numero1 + numero2
resta = numero1 - numero2
multiplicacion = numero1 * numero2
division = numero1 / numero2 if numero2 != 0 else "No se puede dividir entre 0"
porcentaje1 = numero1 * 0.20
porcentaje2 = numero2 * 0.20

print (f"suma: {suma}" )
print (f"resta: {resta}")
print (f"multiplicacion: {multiplicacion}")
print (f"division: {division}")
print (f"El 20% del primer numero: {porcentaje1}")
print (f"El 20% del segundo numero: {porcentaje2}")
